README.md

```diff
@@ -105,7 +105,7 @@ mini-experiments on our code and help us better understand our code and the
 problem we are solving.
 
 ## Instructions
-There are three functions you need to declare and fill in:
+There are three functions have been declared for you. You will need to fill in the following code:
```

Comment:
This is an outstanding commit and the solution is simple *and* also introduces the idea of functions (up next...). Very good stuff.